app/state/index.js

```diff
@@ -1,4 +1,12 @@
 import { combineReducers } from 'redux'
+import StartShiftReducer from './StartShiftReducer'
+
+const appReducer = combineReducers({
+  startShiftReducer: StartShiftReducer,
+})
+
+const rootReducer = (state, action) => {
```

Comment:
Is there going to be an extra purpose for this function, it seems to just wrap appReducer, passing the state and action through. We could simplify this code by renaming `appReducer` to `rootReducer` and just export it directly, without this extra code.

Reply:
done